src/frontend/src/modals/exportModal/index.tsx

```
@@ -53,46 +53,60 @@ const ExportModal = forwardRef(
         size="smaller-h-full"
         open={open}
         setOpen={setOpen}
-        onSubmit={() => {
-          if (checked) {
-            downloadFlow(
-              {
-                id: currentFlow!.id,
-                data: currentFlow!.data!,
+        onSubmit={async () => {
+          try {
+            let filePath: string | undefined | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)